src/files.js

```diff
@@ -90,6 +106,20 @@ function doUpload(cozy, data, method, path, options) {
   })
 }
 
+function addQueryString(path, querystring = {}) {
+  const pathurl = url.parse(path)
+  const query = qs.decode(pathurl.query)
```

Comment:
style (for lines 111-112):

```suggestion
  const query = { ...qs.decode(pathurl.query), ...querystring }
```